features/nanostack/nanostack-hal-mbed-cmsis-rtos/arm_hal_fhss_timer.cpp

```diff
@@ -20,7 +20,7 @@
 #include "ns_types.h"
 #include "fhss_api.h"
 #include "fhss_config.h"
-#include "mbed_trace.h"
+#include "mbed-trace/mbed_trace.h"
```

Comment:
there are  two fixes in this PR like this, is this because `target_include` is not set to include root of these modules ?

Reply:
Can we not changing the files for now, or fix it on master rather and rebase our branch?

Reply:
This is because I want to limit the changes to mbed-trace.
We are working in a fork of mbed-os and mbed-trace actually lives in its own repo, that would/will cause conflict when this fork and mbed-trace will be brought back together in mbed :S